template/extra_git.go

```diff
@@ -0,0 +1,74 @@
+package template
+
+import (
+	"fmt"
+
+	git "gopkg.in/src-d/go-git.v4"
+)
+
+const (
+	gitBase = "Git functons"
+)
+
+var gitFuncs = dictionary{
+	"current_branch": currentBranch,
+	"current_commit": currentCommit,
+	"origin":         origin,
+}
+
+var gitFuncsArgs = arguments{
+	"current_branch": {"path"},
+	"current_commit": {"path"},
+	"origin":         {"path"},
+}
+
+var gitFuncsAliases = aliases{}
+
+var gitFuncsHelp = descriptions{}
```

Comment:
Please add description to maintain the help complete.

Reply:
Oops, forgot to go back and add those :)